### PR TITLE
add swagger docs endpoint

### PR DESCRIPTION
**Pull request by aborilov**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/codegen.sh [hunk 1]
```diff
@@ -142,6 +142,15 @@ statik-gen elevator config/internal/elevator
 $CONTROLLER_GEN rbac:roleName=manager-role paths="./pkg/apiserver/..." output:rbac:artifacts:config=config/internal/apiserver/rbac
 statik-gen apiserver config/internal/apiserver
 
+mkdir tmp_assets
```
[the remark this answers is not in the document]
Comment (aborilov) [reply hack/codegen.sh]: done

**Comment by nmiculinic** [comment]
/lgtm cancel 

the `.pre-commit-config.yaml` is reformated

**Pull request merged**